adapta/storage/database/snowflake_sql.py

```diff
@@ -127,8 +129,9 @@ def publish_external_delta_table(
         database: str,
         schema: str,
         table: str,
-        path: AdlsGen2Path,
-        table_schema: pyarrow.Schema,
+        skip_initialize: bool = False,
```

Comment:
Feel like initialize is a very generic term. Could be anything.
Maybe it's more obvious if you flip the logic, and call it something like this?
```suggestion
        recreate_objects: bool = True,
```

Or if you wanna keep it the logic like it is, then maybe call it something like `refresh_metadata_only` ?